lib/filter-stream.js

```diff
@@ -24,6 +24,7 @@ function FilterStream( blockMap, options ) {
   /** @type {Object} options */
   this.options = options || {}
 
+  this.options.readableObjectMode = true
```

Comment:
Fair enough. I guess if it was better documented, my question would have been redundant ;-)